public/lnd-child-process.js

```diff
@@ -55,16 +57,17 @@ module.exports.startLndProcess = async function({
   lndRestPort,
 }) {
   if (!lndSettingsDir) throw new Error('lndSettingsDir not set!');
+  const rpc = !!rpcUser;
   const processName = 'lnd';
   const args = [
     '--bitcoin.active',
     isDev ? '--bitcoin.simnet' : '--bitcoin.testnet',
-    isDev ? '--btcd.rpcuser=kek' : '',
-    isDev ? '--btcd.rpcpass=kek' : '',
-    isDev ? '--bitcoin.node=btcd' : '--bitcoin.node=neutrino',
-    isDev ? '' : `--configfile=${path.join(lndSettingsDir, 'lnd.conf')}`,
-    isDev ? '' : '--neutrino.connect=btcd0.lightning.engineering',
-    isDev ? '' : '--neutrino.connect=127.0.0.1:18333',
+    isDev || rpcUser ? `--btcd.rpcuser=${rpcUser || 'kek'}` : '',
```

Comment:
Lol, sometimes I wonder about testing these parts of the codebase since it seems like the logic i have to stare at the longest :p